tfhe/src/core_crypto/algorithms/lwe_encryption.rs

```diff
@@ -2245,17 +2847,15 @@ pub fn par_encrypt_lwe_compact_ciphertext_list_with_compact_public_key<
                     },
                 );
 
-                // Noise from Chi_1 for the mask part of the encryption
-                loop_generator
-                    .unsigned_integer_slice_wrapping_add_random_noise_from_distribution_assign(
-                        output_mask.as_mut(),
-                        mask_noise_distribution,
-                    );
+                let zk_body_convolved = polymul_rev(pk_body.as_ref(), binary_random_slice);
+
+                assert_eq!(&pk_body_convolved, &zk_body_convolved);
 
-                // Fill the body chunk afterwards manually as it most likely will be smaller than
-                // the full convolution result. rev(b convolved r) + Delta * m + e2
```

Comment:
I believe we should keep this comment here ?

Reply:
The comment is still here I believe

Reply:
missing the rev(...)